finality-aleph/src/new_network/connection_manager/session.rs

```diff
@@ -342,6 +355,7 @@ mod tests {
         let missing_nodes = handler0.missing_nodes();
         let expected_missing: Vec<_> = (1..NUM_NODES).map(NodeIndex).collect();
         assert_eq!(missing_nodes, expected_missing);
+        assert!(handler0.authentication_for(&NodeIndex(1)).is_none());
```

Comment:
such check is missing from e.g. `ignores_wrong_session_authentication`; as I understand, there is an implicit implication: `x \in missing nodes  => authentication_for(x).is_none()`, but still we should check this anyway

Reply:
Added.